src/main/java/edu/asu/plp/tool/backend/isa/events/MemWatchResponseEvent.java

```diff
@@ -2,26 +2,47 @@
 
 import javafx.beans.property.LongProperty;
 
+/**
+ * This class defines the response object for Memory watch event
+ */
 public class MemWatchResponseEvent {
 	
 	private boolean success;
 	private long watchedAddress;
 	private LongProperty memObject;
 
+	/**
+	 * Constructor for Memory watch response object
+	 * @param success
+	 * @param watchedAddress
+	 * @param memObject
+	 */
 	public MemWatchResponseEvent(boolean success, long watchedAddress, LongProperty memObject) {
 		this.success = success;
 		this.watchedAddress = watchedAddress;
 		this.memObject = memObject;
 	}
 
+	/**
+	 * getter for success flag
```

Comment:
Method `isSuccess` use for indicating the result of memory watch request event. So, the comment getter for success flag won't help the user to understand it. Check how memory watch request been sent out and how the response been handled. E.g. in the handling code (WatchWindow.java line:313).
```
		if (!e.isSuccess())
			return;
```